docs/user/openstack/install_upi.md

```diff
@@ -559,14 +545,13 @@ These ports will always stay `DOWN`. They will never be attached to any Nova ser
 
 There is another fixed IP address (`192.0.2.6`) which will be used for internal DNS records necessary to install and run the OpenShift cluster. These do not need to be exposed to the outside (they only contain internal IPs) and therefore do not need their own Neutron port either. This address will be managed by keepalived as well.
 
-
 #### Assign Floating IPs to the Ports
 
 We have created two floating IP addresses at the beginning. We need to add them to the ports:
 
 ```sh
-$ openstack floating ip set --port "$INFRA_ID-api-port" <203.0.113.23>
-$ openstack floating ip set --port "$INFRA_ID-ingress-port" <203.0.113.19>
+openstack floating ip set --port "$INFRA_ID-api-port" <203.0.113.23>
+openstack floating ip set --port "$INFRA_ID-ingress-port" <203.0.113.19>
```

Comment:
Same here: these can't be copy/pasted so I'd like to leave the shell prompt there.